src/expr/src/relation/func.rs

```diff
@@ -460,7 +460,7 @@ where
     if datums.peek().is_none() {
         Datum::Null
     } else {
-        let x: u64 = datums.map(|d| u64::from(d.unwrap_uint16())).sum();
+        let x: u128 = datums.map(|d| u128::from(d.unwrap_uint16())).sum();
```

Comment:
We might want comments here saying that the value is cast down to a `u64` as part of `reduce`. Though, I'm not sure if that would be too noisy, up to you.

Reply:
Absolutely, better to make things as clear as possible. Comments added!